llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp

```diff
@@ -1784,6 +1899,75 @@ void RISCVInsertVSETVLI::insertReadVL(MachineBasicBlock &MBB) {
   }
 }
 
+static void shrinkInterval(MachineOperand &MO, LiveIntervals *LIS,
+                           const TargetInstrInfo *TII) {
+  Register Reg = MO.getReg();
+  MO.setReg(RISCV::NoRegister);
+  MO.setIsKill(false);
+
+  if (!LIS)
+    return;
+
+  LiveInterval &LI = LIS->getInterval(Reg);
+
+  // Erase the AVL operand from the instruction.
+  SmallVector<MachineInstr *> DeadMIs;
+  LIS->shrinkToUses(&LI, &DeadMIs);
```

Comment:
I think you can use the default nullptr argument
```suggestion
LIS->shrinkToUses(&LI);
```